common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1018,6 +1018,25 @@ private AuthenticationConstants() {
          */
         public static final String BROKER_RESULT_V2 = "broker.result.v2";
 
+        /**
+         * String to return the broker account mode.
+         * This is used to determine what PublicClientApplication MSAL will return to its caller.
+         */
+        public static final String BROKER_ACCOUNT_MODE = "broker.account.mode";
```

Comment:
We recently switched to make these broker constant values snake case (changes in release branch). Can we do the same here for all new constants like `broker.account.mode` to `broker_account_mode